packages/cloud-config/collection.yaml

```diff
@@ -0,0 +1,39 @@
+packages:
+- &base
+  name: defaults
+  category: cloud-config
+  version: "0.1"
+  oem_file: "01_defaults.yaml"
```

Comment:
cant we add the destination in here as well? so in case of change (shouldn't happen) everything is in the same place?